modules/cnv_uninstall_1_4.adoc

```diff
@@ -23,13 +23,17 @@ This procedure uninstalls the following components:
 $ oc login -u system:admin
 ----
 
+. Change directories to `/usr/share/ansible/kubevirt-ansible`:
++
+----
+$ cd /usr/share/ansible/kubevirt-ansible
+----
+
 . Uninstall {ProductName}:
 +
 ----
-$ ansible-playbook -i <inventory file> -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml 
--e @/usr/share/ansible/kubevirt-ansible/vars/cnv.yml 
-/usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml -e 
-apb_action=deprovision
+$ ansible-playbook -i <inventory_file> -e @/usr/share/ansible/kubevirt-ansible/vars/cnv.yml  
```

Comment:
why do you bother to `cd` above if you use the full path here?

Reply:
Nelly had said "also, i believe that we should first cd into /usr/share/ansible/kubevirt-ansible
and execute the command from there in order to not need to be a privileged user" in another comment. That's the only reason I added the `cd` command. But I can shorten those paths and use pushd/popd as you suggested. Will update the PR momentarily.

Reply:
ok. just make sure to do this with `pushd` and finish with `popd` to make @simon3z happy.

Reply:
Please review the changes, @dankenigsberg: https://github.com/openshift/openshift-docs/pull/13328/commits/6010dcf48c2bb4c3d411f0062cb7045bf28be1d2